integration-tests/tests/all_test.go

```diff
@@ -93,7 +93,7 @@ func setUpViamServer(ctx context.Context, t *testing.T) (robot.Robot, error) {
 		"components": [
 			{
 			"name": "%v",
-			"model": "viam:camera:oak-d",
```

Comment:
We will want to keep the old name so we don't migrate - I'd add it to your list of aliased model names.

Is the oak-d a model of oak cameras, or is this a complete typo and we should run a migration script